geocoding/geocode.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import googlemaps
+import os
+
+if not os.path.isdir("Sheets"):
```

Comment:
Parameterize both filename and API key perhaps?

Reply:
Done